common/lib/util/eventemitter.js

```diff
@@ -57,7 +57,7 @@ var EventEmitter = (function() {
 			/* ... or we take event to be the actual event name and listener to be all */
 		}
 		var listeners, idx = -1;
-		if(event === null) {
+		if((event === null) || (event === undefined)) {
```

Comment:
Ok, I will add a commit that works with `(x === null || x === undefined)`.  Is there a helper method that exists anywhere to test this?  If not, it would be useful to have one in the Ably scope.